ckan/requirements.txt

```diff
@@ -6,14 +6,14 @@ boto==2.49.0
 certifi==2021.5.30
 cffi==1.14.6
 chardet==3.0.4
-ckan @ git+https://github.com/ckan/ckan.git@e8b7970423e2cce9731441edd13f7fcfd3be58e9
+ckan @ git+https://github.com/ckan/ckan.git@6731c5a821a6a5f4bdaa20f4e793e0b6ba44f823
 -e git+https://github.com/GSA/ckanext-datagovcatalog.git@64e65702ae1eb5e46d9f37139dc4044c0f253526#egg=ckanext_datagovcatalog
 -e git+https://github.com/GSA/ckanext-datagovtheme.git@29d7454afd516b63cdabb41e7b1dfd4f69daffd4#egg=ckanext_datagovtheme
--e git+https://github.com/GSA/ckanext-datajson.git@34b003239cb80cd5edc758855c3cf6b5e04fbb6a#egg=ckanext_datajson
+-e git+https://github.com/GSA/ckanext-datajson.git@a5d8c9458a7efe955c31b90eeac1e7797881d014#egg=ckanext_datajson
```

Comment:
Datajson still has issues.. but I don't know if those are just accepted for cloud.gov stuff.

Reply:
Accepted, as long as tests pass. I'll make sure to just bump ckan for the FCS branches... :)